app/src/main/java/org/mozilla/fenix/settings/creditcards/controller/CreditCardEditorController.kt

```diff
@@ -66,7 +67,7 @@ class DefaultCreditCardEditorController(
     }
 
     override fun handleDeleteCreditCard(guid: String) {
-        settings.creditCardsDeletedCount += 1
+        metrics.track(Event.CreditCardDelete)
```

Comment:
Can we also add a verify in the tests?

Reply:
I will also add tests. Just wanted to confirm the approach first. This was not marked as ready for review.